### PR TITLE
Upgrade v2 add feature inRow - Fixes #9

### LINKED ISSUE #9: Upgrade to v2
- add feature: _inRow
- add focus highlight around front face items

### PR DESCRIPTION
**Pull request by mayuir2014**
Added feature: _inRow
Added focus highlight around front face items

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,16 +138,4 @@ This component has been tested to the standard Adapt browser specification.
 
 ##Important
 
-This component works in IE9- but has slight change in behaviour, instead of flip effect it gets a fade-in/out effect.
-
 Please feel free to add issues and updates needed in the component.
-
-----------------------------
```
Comment (dancgray) [inline README.md]: Can you add the version number, RTL support etc back in?

**Comment by dancgray** [comment]
The flipcards just scale down in size. there should be a breakpoint to tile the cards vertically, or even replace with narrative.

**Comment by dancgray** [comment]
For accessibility, 
The component does not have aria label describing the component at the start. It needs some description of how the interaction will work for screen readers.

Reads the initial image side of the flipcard OK but does not read the back text. Focus needs to be set in the js file when card flipped

**Comment by mayuir2014** [comment]
Scaling (LTR or RTL) - I tested this on Chrome and IE initially, as well as locally and on the LMS, and all initial tests displayed item boxes stacked correctly as required when scaled down.

What were you testing on? Was it on Safari (Mac/iPad/iPhone)?

**Comment by mayuir2014** [comment]
Ref: Accessibility - I've added the relevant tag info in course.json, but not anywhere else. Did you want to have a chat about this?

**Pull request merged**